cmd/juju/status/output_tabular.go

```diff
@@ -241,20 +261,33 @@ func printApplications(tw *ansiterm.TabWriter, fs formattedStatus) {
 		w.Print(indent("", level*2, name))
 		w.PrintStatus(u.WorkloadStatusInfo.Current)
 		w.PrintStatus(u.JujuStatusInfo.Current)
+
+		printPorts := func(ps []string) {
+			size := len(u.OpenedPorts)
+			for i, op := range u.OpenedPorts {
+				pp := strings.Split(op, "/")
+				w.PrintColorNoTab(output.EmphasisHighlight.BrightMagenta, fmt.Sprintf("%s ", pp[0])) //port e.g 3306
```

Comment:
In out chat discussion decided to remove extra whitespace,

Reply:
Done.